### DISCUSSION TITLE
0.6.0-beta 6

### DISCUSSION THREAD

**Discussion by mdelobelle** [mdelobelle/metadatamenu#428] [Announcements]
Improvements
- Metadata menu options in the property section (with an option to deactivate it)
- fields modal can be called in properties section (manage field at cursor)
- settings UI
- optimised filesFields indexing to apply only on purpose

Fix
- wrong condition check preventing button from showing up
- broken button for fileClass files
- fixed metadata button not triggering the correct file
- Multi and MultiFile not removing indented items before updating

Code cleaning
- indexedDB wrapped code
- separated FieldIndex classes

<hr /><em>This discussion was created from the release <a href='https://github.com/mdelobelle/metadatamenu/releases/tag/0.6.0-beta-6'>0.6.0-beta 6</a>.</em>

**Comment by KuiyueRO** [comment]
Is it possible to support cards in canvas?

This plugin seems to directly use the card id in `.canvas` file as the block id
https://github.com/Quorafind/Obsidian-Canvas-Block-Reference
